### DISCUSSION TITLE
Can I only run comparison algorithm?

### DISCUSSION THREAD

**Discussion by RuolinHe** [antismash/antismash#691] [Q&A]
Hi, I had run antismash for my genomes with mibig comparison, but some of regions weren’t annotated by mibig comparison.

Therefore, I plan to run comparison algorithm against my defined known secondary metabolism BGC datasets. However I don’t want to antismash de novo because it costs a lot of time.

Can I only run comparison algorithm based on existed antismash results?

Thank you!
Ruolin He

**Comment by SJShaw** [comment]
If you already have antiSMASH results and want to add the ClusterCompare comparison analysis, you can do that with a local install of the same version that generated the output originally:

`antismash --reuse <previous_output>.json --cc-mibig`

If you don't have that file from the previous results or don't have previous results at all, then no, you can't do just that analysis. It requires information from previous stages of antiSMASH, which won't otherwise be available.
